### ISSUE TITLE
Something to consider here

### ISSUE THREAD

**Issue by farishkash** [ljhwang/prj-rev-bwfs-tea-cozy#1]
https://github.com/ljhwang/prj-rev-bwfs-tea-cozy/blob/364044af2d2c7deb84161accb68938c6c35ee4ab/TeaCozy/index.html#L33

This should be avoided, did you try padding or margin top/bottom to create spacing?

**Comment by ljhwang** [comment]
Yeah, a number of things I would do if this was "live"  (including alt text). At the end of the day, got to get back to my real job :)
